library/general/containerresources/samples/container-must-have-cpu-requests-memory-limits-and-requests/only-memory-limits-defined-disallowed.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: opa-disallowed
+  labels:
+    owner: me.agilebank.demo
+spec:
+  containers:
+    - name: opa
+      image: openpolicyagent/opa:0.9.2
```

Comment:
Does this work on a more recent version of OPA? We're on version 0.39.0 now:

https://hub.docker.com/layers/opa/openpolicyagent/opa/0.39.0/images/sha256-e6ad6740318e7050de7da895eba5046c040ef11903190d445a4fd1aabe0d3b68?context=explore